### ISSUE TITLE
YSMenu.ini file modifies AUTO_BOOT when using Game Loader in Kernel mode for a specific game

### ISSUE THREAD

**Issue by juniornff** [DS-Homebrew/TWiLightMenu#2499]
**Flashcard or Exploits?** R4i-SDHC RTS 3DS (http://r4i-sdhc.com/)
**Base Console:** NDS Lite 

**TWiLight Menu++ Version:** v27.12.2
**TWiLight Menu++ Theme:** Nintendo 3DS + Persona 3 Reload - S.E.E.S.

### Expected behavior
Console boots into TWiLight Menu

### Actual behavior
Console boots into the game through YSMenu
[Video Demostration](https://youtube.com/shorts/xsWd21L6LqA)

### Steps to reproduce
1. Set Game Loader to Kernel for a specific game
2. Launch the game
3. Reset the console

### Notes
i'm trying to play a patched version of _Chou Soujuu Mecha MG_ and using nds-bootstrap freeze the game after the first text introduction. When I use the Kernel it works, but the `YSMenu.ini` file changes the value of `AUTO_BOOT` from `BOOT.NDS` to `/roms/nds/Chou Soujuu Mecha MG (Japan) (patched).nds` so when I turn the console off and on again I have to use the button combination `(A+B+X+Y+L+R)` to return to TwilightMenu. i have a copy of `YSMenu.ini` so i can set it back, but it's quite annoying

YSMenu.ini [before](https://github.com/user-attachments/files/18167165/YSMenu-before.txt) and [after](https://github.com/user-attachments/files/18167172/YSMenu-after.txt)

**Comment by Deletecat** [comment]
This is not a bug. This is how Twilight Menu is able to load games through YSMenu. 

If you want to auto-boot Twilight Menu while also being able to use YSMenu as a game loader, you should make use of the autoboot files provided in Twilight Menu's flashcard package. Using the autoboot files means that you will always boot into Twilight Menu first, regardless of what `AUTO_BOOT` is set to in your `YSMenu.ini` config.
For your flashcard, you can find them in the `Autoboot/R4i-SDHC, r4isdhc.com cards 2014+, R4i SDHC Upgrade Revolution, R4DSiXL3D, R4i Advance, R4-IIIi, R4 SDHC Revolution, R4(i) Pocket, R4i Gold (v1.4.1) (3DS) & R4xDS` folder within the `TWiLightMenu-Flashcard.7z` package. 
Simply put `R4.dat` and `TTMenu.dat` onto the root of your microSD card, replacing the files if asked. After that, you should be able to consistently auto-boot into Twilight Menu.

Setting `AUTO_BOOT`  to `BOOT.NDS` was a hacky method used for specific DSTT clones that could boot into YSMenu, but couldn't boot into Twilight Menu using the provided autoboot files. This shouldn't be an issue now as of release v27.12.0
